src/main/java/edu/hawaii/its/api/controller/ForTestingRestController.java

```diff
@@ -11,19 +16,51 @@
 
 import edu.hawaii.its.api.exception.ExceptionForTesting;
 
+import java.util.List;
+
 @RestController
 @RequestMapping("/api/groupings/v2.1/testing")
 public class ForTestingRestController {
 
     private static final Log logger = LogFactory.getLog(ForTestingRestController.class);
 
+    @Autowired
+    GroupingPropertiesService groupingPropertiesService;
+
     @GetMapping(value = "/exception")
     @ResponseBody
     public ResponseEntity throwException() {
         logger.debug("Entered REST throwException");
-            ExceptionForTesting exception = new ExceptionForTesting("Exception for test failed")
+        ExceptionForTesting exception = new ExceptionForTesting("Exception for test failed")
                 .addSubError(new ApiValidationError("testing object 1", "membership service", "id: 12", "Membership access denied"))
                 .addSubError(new ApiValidationError("testing object 2", "member service", "id: 30", "There is no member"));
         throw exception;
     }
+
+//    @GetMapping(value = "/member")
+//    @ResponseBody
+//    public ResponseEntity<GroupingMember> memberBeanTest() {
+//        logger.debug("Entered member Bean Test");
+//        return ResponseEntity
+//                .ok()
+//                .body(groupingPropertiesService.getGroupingMemberBean());
+//    }
+//
+//    @GetMapping(value = "/members")
+//    @ResponseBody
+//    public ResponseEntity<List<GroupingMember>> membershipsBeanTest() {
+//        logger.debug("Entered memberships Bean Test");
+//        return ResponseEntity
+//                .ok()
+//                .body(groupingPropertiesService.getGroupingMembersBean());
+//    }
+//
+//    @GetMapping(value = "/hasMember")
+//    @ResponseBody
+//    public ResponseEntity<HasMembersResults> hasMemberResultBeanTest() {
+//        logger.debug("Entered memberships Bean Test");
+//        return ResponseEntity
+//                .ok()
+//                .body(groupingPropertiesService.getHasMembersResultsBean());
+//    }
```

Comment:
Make sure to remove this comment or leave a TODO: comment over it if you plan on leaving this in.